.github/workflows/ci.yml

```diff
@@ -0,0 +1,21 @@
+name: Node.js Test
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
```

Comment:
This repo uses `main` because Github decided to change the default for some reason.

I can either change it to `master` (but then the PR would need to be resubmitted), or this needs to be updated. Feel free to make the call.

Reply:
Indeed.

Reply:
BTW, note that you can adjust the name of the default branch on your GitHub preferences.